src/state/__tests__/state-machine/borrow.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test, expect } from "vitest";
+import { getTestActor } from "@/state/__tests__/utils";
+import { map } from "lodash";
+
+test("can unborrow resource card", async () => {
+  const { send, getState } = getTestActor();
+
+  const stateBefore = getState();
+
+  send({ type: "user.click.market.deck.element", name: stateBefore.elementMarket.deck[0].name });
+
+  const stateAfterFirstBorrow = getState();
+
+  send({ type: "user.click.market.deck.element", name: stateAfterFirstBorrow.elementMarket.deck[0].name });
+
+  const stateAfterSecondBorrow = getState();
+
+  expect(stateAfterSecondBorrow.turn.borrowedElement).not.toEqual(stateAfterFirstBorrow.turn.borrowedElement);
+});
+
+test("can't borrow twice", async () => {
```

Comment:
This test is not valid anymore right? We confirmed that it is in fact possible to borrow more than once?

Reply:
True.. this one actually testing other thing.
I'll change description.